modules/ROOT/partials/integrations/svelte-tech-ref.adoc

```diff
@@ -13,7 +13,7 @@ The `+tinymce-svelte+` `+Editor+` component accepts the following properties:
 ----
 <Editor
   apiKey="api-key"
-  channel="5"
+  channel="6"
```

Comment:
This might be better using `{productmajorversion}` here instead of `6`, as then it won't need manually updating. If you do that though you'll need to make sure attribute substitution is enabled for the code snippet though. e.g `[source,jsx,subs="attributes+"]`.